src/isPalindrome.js

```diff
@@ -0,0 +1,11 @@
+export default function isPalidrome (str){
+  var re = /[^A-Za-z0-9]/g;
+  str = str.toLowerCase().replace(re, '');
+  var len = str.length;
```

Comment:
Since the `length` of the string does not change during execution, consider using `str.length` inline.  Since it's used a few times, I can see the case for this variable, but if you would like to keep it, consider giving it a more intention revealing name: `stringLength`, or even `length`.